### DISCUSSION TITLE
Select New Page Location

### DISCUSSION THREAD

**Discussion by aisied** [requarks/wiki#3068] [Help / Questions] [answered]
After giving permissions to a specific folder for a department, the user can't select a folder with creating a new page.  
![image](https://user-images.githubusercontent.com/26960924/107757532-5f510300-6d2e-11eb-94a1-4b35a1aa729b.png)

**Comment by NGPixel** [comment, marked as answer]
You can still write the path in the text field. The directory listing won't work as the user doesn't have permissions to list them.

So if you gave access to `/sub/folder`, simply write `/sub/folder/new-page` in the text field and click Select.
